contributing/code/standards.rst

```diff
@@ -101,33 +109,34 @@ short example containing most features described below::
             );
 
             if (true === $dummy) {
-                return null;
+                return 'something';
```

Comment:
Note: alternatively you could keep `return null;` here and do `return 'something';` at the end, but feels "weird" to me